### ISSUE TITLE
Is such combining with apache 2.0 license is correct?

### ISSUE THREAD

**Issue by MBkkt** [jothepro/doxygen-awesome-css#109]
https://github.com/jothepro/doxygen-awesome-css/blob/main/LICENSE

https://github.com/jothepro/doxygen-awesome-css/blob/main/doxygen-awesome-darkmode-toggle.js#L31

**Comment by tim-gromeyer** [comment]
I have read following 2 sentences:

> The MIT and Apache 2.0 licenses are both permissive licenses that put no or very little restriction on what licenses you can use for your own code (even if that code is a modification to MIT/Apache 2.0 licensed code).

Source: https://opensource.stackexchange.com/a/12904/25784

> Both the Apache License and the MIT license are permissive, so incorporating MIT licensed code into your Apache licensed project is certainly allowed. Just be sure to attribute the original author for the parts your incorporated and include a copy of the MIT License terms, as required by the license.

Source: https://law.stackexchange.com/a/6758

I'm definitely not a licensing expert but according to quotes I put above it's possible as long as you `attribute the original author for the parts your incorporated and include a copy of the MIT License terms, as required by the license`.

**Comment by MBkkt** [comment]
@tim-gromeyer 

> Both the Apache License and the MIT license are permissive, so incorporating MIT licensed code into your Apache licensed project is certainly allowed. 

Yes, it's definitely true, mit can be used almost everywhere, mit a little less restrictive than apache2.


But this issue is about is it ok to use apache2 in mit

**Comment by MBkkt** [comment]
> Both the Apache License and the MIT license are permissive, so incorporating MIT licensed code into your Apache licensed project is certainly allowed

> incorporating MIT licensed code into your Apache licensed project

Here is opposite situation, it's reason why I created an issue. I also not license expert, so...

**Comment by tim-gromeyer** [comment]
I think the best thing we can do is to put it on [Quora](https://www.quora.com).

**Comment by a4z** [comment]
The Google icons are also under the Apache license.
At least those from this repo:
https://github.com/google/material-design-icons

so maybe everything is just fine?

**Comment by jothepro** [comment]
Mh, I think I am starting to understand the problem space. I have explicitly *mentioned* the source of the icons and I have *referenced* the License in the code file:

```js
// SVG icons from https://fonts.google.com/icons
// Licensed under the Apache 2.0 license:
// https://www.apache.org/licenses/LICENSE-2.0.html
static lightModeIcon = `<svg xmlns="http://www.w3.org/2000/svg"...`
static darkModeIcon = `<svg xmlns="http://www.w3.org/2000/svg"...`
```

from my understanding this provides both the required *notice* and *disclaimer*, but I didn't *include* a copy of the License. So adding a copyright folder in addition to the link to the license might indeed make sense. 👍

**Comment by a4z** [comment]
I also wonder what that means when using [doxygen-awesome-css](https://github.com/jothepro/doxygen-awesome-css), 
I guess something needs to be done ... 🤔

**Comment by MuellerSeb** [comment]
What about "drawing" a moon and a sun "by hand" and add it to the repo? svg could also be inlined, right?
